generator/src/test/java/com/graphicsfuzz/generator/semanticschanging/Expr2BinaryMutationFinderTest.java

```diff
@@ -80,11 +80,26 @@ public int nextInt(int bound) {
         return 2;
       }
 
+      @Override
+      public int nextInt(int origin, int bound) {
+        return IRandom.super.nextInt(origin, bound);
+      }
+
+      @Override
+      public long nextLong(long bound) {
+        return 2;
+      }
+
       @Override
       public Float nextFloat() {
         throw new UnsupportedOperationException();
       }
 
+      @Override
+      public int nextPositiveInt(int bound) {
+        return IRandom.super.nextPositiveInt(bound);
+      }
+
```

Comment:
Hmm, I might be missing something but I'm having trouble understanding why this wasn't required before your changes but is now.